deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py

```diff
@@ -257,14 +257,17 @@ def check_provider_database_update_start_log(provider_name, timeout=vd.VULN_DETE
                               error_message=f"Could not find {provider_name} update starting log")
 
 
-def check_provider_database_update_finish_log(provider_name, timeout=vd.VULN_DETECTOR_GLOBAL_TIMEOUT):
+def check_provider_database_update_finish_log(wazuh_log_monitor, provider_name='',
```

Comment:
```suggestion
def check_provider_database_update_finish_log(wazuh_log_monitor=None, provider_name='',
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2575/commits/73dd596fd88d841d166213eb00ccae04e87dc800

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2575/commits/73dd596fd88d841d166213eb00ccae04e87dc800